app/src/main/java/com/kickstarter/viewmodels/ProjectSearchResultHolderViewModel.java

```diff
@@ -18,113 +18,102 @@
 
 public interface ProjectSearchResultHolderViewModel {
 
-  final class Data {
```

Comment:
This is fine. I still prefer data structures to pairs, but I love me less lines of code, too!